src/sage/arith/misc.py

```diff
@@ -1955,7 +1955,7 @@ def xlcm(m, n):
     return (l, m, n)
 
 
-def xgcd(a, b):
+def xgcd(a, b=None):
     r"""
     Return a triple ``(g,s,t)`` such that `g = s\cdot a+t\cdot b = \gcd(a,b)`.
```

Comment:
Should this be updated?

Reply:
I think so, I also checked locally without update and gives `TypeError: xgcd() missing 1 required positional argument: 'b'` without assigning default value.

Reply:
I meant the documentation here, i.e., the first line.